packages/docusaurus-plugin-content-blog/src/__tests__/feed.test.ts

```diff
@@ -62,6 +68,32 @@ async function testGenerateFeeds(
   });
 }
 
+function isFullAbsolutePath(str: string) {
+  const domain = 'https://domain.com';
+  const {origin} = new URL(str, domain);
+  return origin !== domain;
+}
+
+async function generateLinksOfBlogPosts(outDir: string, blogPosts: BlogPost[]) {
+  const linksOfBlogPosts: {[postId: string]: string[]} = {};
+  const pathOfFile = path.join(outDir, 'blog');
+  const promises = blogPosts.map(async (post) => {
+    try {
+      const content = await readOutputHTMLFile(post.id, pathOfFile, true);
+      const $ = cheerioLoad(content);
+      const anchorElements = $(`div#${blogPostContainerID} a`);
+      if (anchorElements.length > 0) {
+        const href = anchorElements.map((_, elm) => elm.attribs.href).toArray();
+        linksOfBlogPosts[post.id] = href;
+      }
+    } catch {
+      // post is a draft
+    }
+  });
+  await Promise.all(promises);
+  return linksOfBlogPosts;
+}
```

Comment:
I don't think we need that complexity inr our tests

Reply:
The test case is removed